features/themes/onms-default-theme/bin/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm, I'm not sure what these three files are. I'll have to remove them from the PR.